zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -309,7 +301,8 @@ pub(crate) struct AssignedECDSA<F: Field> {
 #[derive(Debug)]
 pub(crate) struct AssignedSignatureVerify<F: Field> {
     pub(crate) address: AssignedValue<F>,
-    pub(crate) msg_hash_rlc: AssignedValue<F>,
+    pub(crate) msg_hash_lo: AssignedValue<F>,
+    pub(crate) msg_hash_hi: AssignedValue<F>,
```

Comment:
I suggest replacing this by:
```
pub(crate) msg_hash: Word<AssignedValue<F>>,
```

Reply:
fixed [cf7e444](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1418/commits/cf7e444991feb62ad14d93754d25989abd30ef57)